pkg/controllers/updaterun/execution.go

```diff
@@ -191,6 +206,7 @@ func (r *Reconciler) executeUpdatingStage(
 			return 0, err
 		}
 		if approved {
+			unmarkUpdateRunWaiting(updateRun)
```

Comment:
ditto

Reply:
Same as above, the after-stage task completes, we are sure that it's not waiting anymore, and we remove that.